src/js/rtc_const.js

```diff
@@ -34,13 +34,19 @@ export const BYE_METHOD_NAME = "bye";
 export const RTC_PEER_CONNECTION_CONFIG = {
     iceTransportPolicy: 'relay',
     rtcpMuxPolicy: 'require',
-    bundlePolicy: 'balanced'
+    bundlePolicy: 'balanced',
```

Comment:
How do these configs affect non-citrix calls? Why are enforcing this on all calls?

Reply:
It won't affect non-citrix calls.
The non-citrix calls have the same value for these fields by default.
bundlePolicy is `balanced` by default. https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection#bundlepolicy
sdpSemantics is `unified` by default https://webrtc.org/getting-started/unified-plan-transition-guide#phase_3_switch_the_default
enableDtlsSrtp is `true` by default